src/main/webapp/scripts/nearbyVendors.js

```diff
@@ -29,7 +38,7 @@ async function querySalecard() {
   }
 
   // Draw client information.
-  const map = drawMap(clientInfo);
+  map = drawMap(clientInfo);
```

Comment:
Just one question, is this a global variable? And in which file is it declared?

Reply:
Same question, don't we need to do 'const map...'? Or at least 'let map...'?